abigen-tests/src/abi/tests.rs

```diff
@@ -542,7 +542,7 @@
                                     .as_slice(),
                             ),
                         ),
-                        ethabi::Token::Bool(self.param6),
+                        ethabi::Token::Bool(self.param6.clone()),
```

Comment:
Let's move that to another PR (I can tackle the next PR for it also, I was planning to fix the `bool[]` reported on Slack anyway).